tests/system/test_setup.py

```diff
@@ -0,0 +1,78 @@
+import shutil
+import os
+import unittest
+
+from elasticsearch import Elasticsearch
+from apmserver import BaseTest
+from beat.beat import INTEGRATION_TESTS
+
+
+class SetupTemplateDefaultTest(BaseTest):
+    """
+    Test setup template subcommand with default option.
+    """
+
+    def get_elasticsearch_url(self):
+        """
+        Returns an elasticsearch.Elasticsearch instance built from the
+        env variables like the integration tests.
+        """
+        return "http://{host}:{port}".format(
+            host=os.getenv("ES_HOST", "localhost"),
+            port=os.getenv("ES_PORT", "9200"),
+        )
+
+    def setUp(self):
+        super(BaseTest, self).setUp()
+
+        self.elasticsearch_url = self.get_elasticsearch_url()
+        self.es = Elasticsearch([self.elasticsearch_url])
+
+        try:
+            self.es.indices.delete_template('apm-*')
+        except:
+            pass
+        assert self.es.indices.exists_template(name='apm-*') == False
```

Comment:
not a big deal, but `assert not self.es.indices.exists_template(name='apm-*')` would be more idiomatic imo